wordle-web/components/game-board.vue

```diff
@@ -1,8 +1,8 @@
 <template>
   <v-card width="450">
-    <v-container>
+    <v-container class="container-gradient">
       <v-row v-for="row in wordleGame.maxGuesses" :key="row" dense>
-        <v-col v-for="index in wordleGame.currentWord.maxLetters" :key="index">
+        <v-col v-for="index in maxLetters" :key="index">
           <v-card height="50" :color="letterColor(getLetter(row, index))">
             <v-card-text class="text-center">
```

Comment:
The letters on the gameboard are a little hard to read, here is some styles you could try:
<v-card-text class="text-h5 font-weight-bold">